tests/test_apidoc.py

```diff
@@ -118,6 +120,15 @@ def test_pep_0420_disabled_top_level_verify(make_app, apidoc):
     print(app._status.getvalue())
     print(app._warning.getvalue())
 
+@pytest.mark.apidoc(coderoot=(rootdir / 'root' / 'trailing_underscore'))
```

Comment:
Thanks, moving it is very helpful for us! :-)

>In order to verify #519, I think yes, it does need to be built. But I guess that can wait until that test exists. And no, you're right - not with all builders.

I think we can verify it through checking generated reST notation.
But, as you said, it might be needed. I don't have strong opinion for that.

We can test it with following notation:
```
@pytest.mark.apidoc(coderoot=(rootdir / 'roots' / 'test-apidoc-underscore'))
def test_apidoc_underscore(make_doc, apidoc):
    app = make_app('dummy', srcdir=outdir)
    app.build()
    print(app._status.getvalue())
    print(app._warning.getvalue())
```

note: `dummy` builder does not generate anything. It only parses documents. In this case, it might work well.